pytest_html/plugin.py

```diff
@@ -235,9 +271,10 @@ def _generate_report(self, session):
         if PY3:
             self.style_css = self.style_css.decode('utf-8')
 
-        html_css = html.link(href='style.css', rel='stylesheet',
+        css_href = os.path.join('assets', 'style.css')
```

Comment:
This needs to be joined by `/` and not the system path separator, otherwise it will fail on Windows.